src/lib.rs

```diff
@@ -596,11 +596,11 @@ impl NetmapDescriptor {
         return None;
     }
 
-    #[allow(dead_code)]
     pub fn get_fd(&self) -> i32 {
         self.desc.get().fd
     }
 
+    #[deprecated]
```

Comment:
Sure - you can definitely do that too - still can, but I'd put the method on your thread object perhaps? Anyhow - do you want me to back this part of the change out?